Dockerfile

```diff
@@ -0,0 +1,71 @@
+FROM ruby:2.7.7
+
+ARG RAILS_ENV
+ARG SECRET_KEY_BASE
+
+# Necessary for bundler to operate properly
+ENV LANG C.UTF-8
+ENV LC_ALL C.UTF-8
+
+# add nodejs and yarn dependencies for the frontend
+# RUN curl -sL https://deb.nodesource.com/setup_14.x | bash - && \
+#  curl -sS https://dl.yarnpkg.com/debian/pubkey.gpg | apt-key add - && \
+#  echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources.list.d/yarn.list
+
+# --allow-unauthenticated needed for yarn package
+RUN apt-get update && apt-get upgrade -y && \
+  apt-get install --no-install-recommends -y ca-certificates nodejs \
+  build-essential libpq-dev libreoffice imagemagick unzip ghostscript vim \
+  libqt5webkit5-dev xvfb xauth default-jre-headless --fix-missing --allow-unauthenticated
+
+RUN apt-get update && apt-get install -y wget gnupg
+RUN apt-get install -y wget apt-transport-https gnupg
+RUN echo "deb https://packages.adoptium.net/artifactory/deb $(awk -F= '/^VERSION_CODENAME/{print$2}' /etc/os-release) main" | tee /etc/apt/sources.list.d/adoptium.list
+
+RUN wget -qO - https://packages.adoptium.net/artifactory/api/gpg/key/public| apt-key add - \
+    && apt-get update \
+    && apt-get install -y temurin-8-jdk
+
+RUN update-java-alternatives --set /usr/lib/jvm/temurin-8-jdk-amd64
```

Comment:
I suspect we may start needing this change in other places, worth noting: https://adoptium.net/blog/2023/07/adoptopenjdk-jfrog-io-has-been-deprecated/